.github/renovate.json

```diff
@@ -7,17 +7,53 @@
     {
       "packagePatterns": ["*"],
       "excludePackagePatterns": [
+        "@babel/core",
+        "@babel/plugin-proposal-class-properties",
+        "@babel/plugin-proposal-decorators",
+        "@babel/plugin-proposal-object-rest-spread",
+        "@babel/plugin-proposal-optional-chaining",
+        "@babel/plugin-proposal-private-methods",
+        "@babel/plugin-proposal-private-property-in-object",
+        "@babel/preset-env",
+        "@babel/preset-react",
+        "@babel/preset-typescript",
+        "@octokit/rest",
         "@patternfly/patternfly",
         "@patternfly/patternfly-a11y",
-        "@patternfly/documentation-framework"
+        "@patternfly/documentation-framework",
+        "@testing-library/jest-dom",
+        "@types/jest",
+        "@typescript-eslint/eslint-plugin",
+        "@typescript-eslint/parser",
+        "eslint",
+        "lint-staged",
+        "prettier"
       ],
       "enabled": false
     },
     {
       "datasources": ["npm"],
       "packageNames": [
```

Comment:
If we do the above suggestion I think we would have to change this key to "packagePatterns" to condense these included packages.